local-k8s.sh

```diff
@@ -55,11 +55,18 @@ if [[ $DEBUG == "true" ]]; then
     set -x
 fi
 
-if [[ "$ENABLE_BACKUP" == "true" ]] || [[ "$S3_OFFLOAD" == "true" ]] || [[ "$USAGE_S3" == "true" ]]; then
-    need_minio="true"
-else
-    need_minio="false"
-fi
+function verify_minio_config(){
+    IFS='.' read -r major_v minor_v patch_v <<< "$WEAVIATE_VERSION"
```

Comment:
There is a slight problem with this, as sometimes we use the preview tags, for example: "[preview-spfresh-add-as-vector-indexes-option-d140d1f.amd64](https://hub.docker.com/layers/semitechnologies/weaviate/preview-spfresh-add-as-vector-indexes-option-d140d1f.amd64/images/sha256-f4b933a6e9dcf5e727b77540f9441de55c063d11d998bfcad52b8c06bcab1d0b)" How will local-k8s behave when the version passed has that format instead of a semver one?